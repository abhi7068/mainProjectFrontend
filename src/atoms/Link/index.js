import React from 'react';

import Typography from '@material-ui/core/Typography';
import PropTypes from 'prop-types';

function LinkComponent({ children }) {
  return (
    <Typography data-testid="hello" variant="body2" mx={20}>
      {children}
    </Typography>
  );
}
LinkComponent.propTypes = {
  children: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
};
export default LinkComponent;
