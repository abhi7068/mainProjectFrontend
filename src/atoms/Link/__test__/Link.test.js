import React from 'react';

import Link from '../index';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<Link />);
});
it('should take a snapshot', () => {
  const { asFragment } = render(<Link />);
  expect(asFragment(render(<Link />))).toMatchSnapshot();
});
