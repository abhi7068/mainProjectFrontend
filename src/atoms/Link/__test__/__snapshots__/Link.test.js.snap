// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body2"
    data-testid="hello"
    mx="20"
  />
</DocumentFragment>
`;
